src/main/resources/org/jenkinsci/plugins/ghprb/GhprbTrigger/global.groovy

```diff
@@ -46,7 +46,7 @@ f.section(title: descriptor.displayName) {
     f.entry(field: "cron", title: _("Crontab line"), help: "/descriptor/hudson.triggers.TimerTrigger/help/spec") {
       f.textbox(default: "H/5 * * * *", checkUrl: "'descriptorByName/hudson.triggers.TimerTrigger/checkSpec?value=' + encodeURIComponent(this.value)") 
     }
-    f.entry(field: "labelslist", title: _("List of github labels for which the build should not be triggered.")) {
+    f.entry(field: "labelsIgnoreList", title: _("List of github labels for which the build should not be triggered.")) {
```

Comment:
As per your point on the fix you implemented, could you point out that this list needs to have carriage return characters as separators?